recipes-extended/luci/luci_git.bb

```diff
@@ -18,10 +18,10 @@ inherit cmake openwrt pkgconfig
 
 prefix=""
 includedir="/usr/include"
+bindir="usr/bin"
```

Comment:
As noted is necessary because prefix="" and needs to be relative because of they way CMake handles absolute vs relative paths (verified experimentally).